components/Report/index.tsx

```diff
@@ -0,0 +1,3 @@
+export { default as Cards } from './Cards';
+export { default as RangeSection } from './RangeSection';
+export { default as Summary } from './Summary';
```

Comment:
Don't need use default. Can use `export Summary from './Summary';`

Reply:
Don't think that works

Reply:
let merge in and see